commerce-product-service/src/main/java/com/liferay/commerce/product/internal/importer/CPFileImporterImpl.java

```diff
@@ -763,6 +774,40 @@ protected void updateActions(
 		}
 	}
 
+	protected void updatePermission(
```

Comment:
Hey @Commando18, can you update how the logic is sorted within this method? Since variables should always be declared just before they're needed (unless there's a strong reason otherwise), I think it should look like:

1) `int scope`
2) `String roleName`
3) `Role role`
4) `String[] actionIds`
5) `JSONArray actionIdsJSONArray`
6) use `actionIdsJSONArray`
7) `_resourcePermissionLocalService.setResourcePermissions(...` (which uses `scope`, `role`, and `actionIds`)

Please let me know if you have any questions what I mean!